File: atividade1/atividade1.py

class Aluno:
    def __init__(self, ra, nome):
        self.ra = ra
        self.nome = nome

    def imprimir_info(self):
        print(f"RA: {self.ra}")
        print(f"Nome: {self.nome}")


class MeuAluno(Aluno):
    def verifica_ra(self):
        meu_ra = "N019395"
        if self.ra == meu_ra:
            print("Este RA pertence a você!")
        else:
            print("Este RA não pertence a você.")

ra_inserido = input("Insira seu RA: ")
nome_inserido = input("Insira seu nome: ")


meu_aluno = MeuAluno(ra_inserido, nome_inserido)


meu_aluno.imprimir_info()


meu_aluno.verifica_ra()

File: atividade1/atividade1.js
class Aluno {
  constructor(ra, nome) {
    this.ra = ra;
    this.nome = nome;
  }

  imprimirInfo() {
    console.log(`RA: ${this.ra}`);
    console.log(`Nome: ${this.nome}`);
  }
}

class MeuAluno extends Aluno {
  verificaRA() {
    const meuRA = "N019395";
    if (this.ra === meuRA) {
      console.log("Este RA pertence a você!");
    } else {
      console.log("Este RA não pertence a você.");
    }
  }
}

const raInserido = prompt("Insira seu RA:");
const nomeInserido = prompt("Insira seu nome:");

const meuAluno = new MeuAluno(raInserido, nomeInserido);

meuAluno.imprimirInfo();

meuAluno.verificaRA();
